src/main/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -88,8 +88,12 @@
             "</pre>" +
             "<hr />";
     public static interface ProgramInterface {
+        default  SinglePassSamProgram makeInstance(final String outbase, final String outext, final File input, final File reference,
```

Comment:
Can you comment that the default implementations "forgets" the value of `includeUnpaired`, and calls the version of makeInstance that doesn't use it? It's clear if you read it carefully, but can be missed by a casual observer.